service.subtitles.torec/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.subtitles.torec"
        name="Torec"
-       version="1.1.4"
+       version="1.2.0"
        provider-name="slmosl">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Can you adjust the python version to match the one for gotham? It should be  2.14.0